cookbook/profiler/data_collector.rst

```diff
@@ -46,106 +46,160 @@ access to in local properties.
     store objects that cannot be serialized (like PDO objects), or you need
     to provide your own ``serialize()`` method.
 
-Most of the time, it is convenient to extend
+Most of the times, it is convenient to extend
```

Comment:
After a bit googling, I find many people saying that it should always be singular. Apart from [this one]():

 > **Most of the time** means *usually or very often*; **most of the times** is a bit different and more specific, meaning *almost every time (when); almost at every single attempt*.

Using this definition, it seems like plural is correct here, but I would stick with time.